src/navigator/AppNavigator.js

```diff
@@ -8,19 +8,21 @@ import InProgressComplaint from '../complaints/component/InProgressComplaint';
 import HomeScreen from '../screens/HomeScreen';
 import SingleComplaint from '../complaints/component/SingleComplaint';
 import ComplaintTracking from '../complaints/ComplaintTracking';
+import Welcome from '../screens/Welcome';
 
 const Stack = createNativeStackNavigator();
 
 const AppNavigator = () => {
   return (
-    <Stack.Navigator initialRouteName='signin'>
+    <Stack.Navigator initialRouteName='signup'>
       <Stack.Screen name="signin" component={Signin} options={{headerShown:false}}/>
       <Stack.Screen name='signup' component={SignUp} options={{headerShown:false}}/>
       <Stack.Screen name='home' component={HomeScreen} options={{headerShown:false}} />
       <Stack.Screen name='Recieved' component={RecievedComplaint} options={{headerShown:false}} />
       <Stack.Screen name='complaint' component={ComplaintScreen} options={{headerShown:false}} />
       <Stack.Screen name='single' component={SingleComplaint} options={{headerShown:false}} />
       {/* <Stack.Screen name='tracking' component={ComplaintTracking} /> */}
+      <Stack.Screen name='welcome' component={Welcome} options={{headerShown:false}} />
     </Stack.Navigator>
   );
```

Comment:
Great!! I suggest keeping all the private paths for authenticated users and auth screens for unauthenticated users. It is our initial step and it is better for now, once we integrate the API we should do this first... please R & D this thing for now.